### DISCUSSION TITLE
How to stop putting in bot token every time?

### DISCUSSION THREAD

**Discussion by 77Whitehawk77** [jagrosh/MusicBot#1461] [Questions]
Every time I run JMusicBot, I have prompts that ask me to fill in the bot token and then owner ID. I have it already in the config file, so why is it asking every time I run it?

**Comment by jagrosh** [comment]
It's not using the config file you've set up. Check the log output to see where it says it's reading the config from. Also make sure that you're on a platform where writing persistent files is possible.

**Reply by 77Whitehawk77** [reply]
I'm not much of a coder. How do I find the log output? And what platform should I use?

**Reply by 77Whitehawk77** [reply]
Not sure how to proceed.

**Reply by jagrosh** [reply]
How are you running the bot, and what does it print out (in the Console) when it starts up?

**Reply by 77Whitehawk77** [reply]
![Screenshot (41)](https://github.com/jagrosh/MusicBot/assets/153327709/f9a40bf4-aae6-4761-a7bd-7d48ef96b018)

**Reply by 77Whitehawk77** [reply]
This is what comes up. When I type in the whole token, it then asks me for my owner ID.

**Reply by 77Whitehawk77** [reply]
And when I exit, it shows that the config file location is exactly where it should be.

**Reply by jagrosh** [reply]
After you enter a token and owner ID, what is printed in the console?

**Reply by 77Whitehawk77** [reply]
So it loads, and it says "Finished Loading". The problem is having to type in the token and owner ID every time

**Reply by jagrosh** [reply]
Can you share the entire output of the Console

**Reply by 77Whitehawk77** [reply]
![Screenshot (42)](https://github.com/jagrosh/MusicBot/assets/153327709/02b84476-a9c4-4d57-8d91-8afd614bfbc8)

**Reply by jagrosh** [reply]
When it prompts for a token, if you close the window (without entering anything), what path does it show you?

**Reply by 77Whitehawk77** [reply]
When I close the window, nothing happens, it just closes.

**Reply by 77Whitehawk77** [reply]
Then I open it again, and it gives the same prompts.

**Reply by jagrosh** [reply]
Or just click 'OK' without entering a token, what is the output?

**Reply by 77Whitehawk77** [reply]
It has me add the owner ID, but then it doesn't work.

**Reply by jagrosh** [reply]
What if you hit Cancel when entering a token?

**Reply by 77Whitehawk77** [reply]
Then it exits out and basically says try again.

**Reply by jagrosh** [reply]
Can you screenshot the output after you hit cancel
